### PR TITLE
Add resolution feature

### PR DESCRIPTION
**Pull request by kfbfarley**
## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] The commit message follows our guidelines: https://github.com/ngneat/falso/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [x] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Other... Please describe:

## What is the current behavior?

- Issue Number: N/A
- Resolve #issueNumber

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

**Comment by kfbfarley** [comment]
@NetanelBasal just lemme know if there's any problem with this, you can just ignore **.devcontainer** if you don't find it interesting

File: packages/falso/src/lib/resolution.json [hunk 1]
```diff
@@ -0,0 +1,436 @@
...
+    {
+      "width": "1080",
+      "height": "2400"
+    },
+    {
+      "width": "1440",
+      "height": "3200"
+    },
+    {
+      "width": "1440",
```
Comment (NetanelBasal) [inline packages/falso/src/lib/resolution.json:360]: - You have duplicate data here
- Let's start with 5-10 top

Comment (kfbfarley) [reply packages/falso/src/lib/resolution.json:360]: that was a good one, i didn't notice, will fix asap

**Comment by kfbfarley** [comment]
> What is devcontainer?

You can have a look [here](https://code.visualstudio.com/docs/devcontainers/containers).

You can run your project inside a container with no need to install node in your local magine, everything is installed in the container using a **[Dockerfile](https://github.com/ngneat/falso/blob/5935f662919e05e91b3b724c24ce7d4fd93adcfd/.devcontainer/Dockerfile)**, when you open vscode for the first time or with **.devcontainer** changes it asks you to *Open in Container* and the projects loads your project files to the container and completes the packages and services installation.

Basically the development space is the same for everyone who's using DevContainers.

Please, let me know if you interested in adding devcontainers to your project.

- [ ] Yes
- [ ] No

I will update the PR asap.

**Comment by kfbfarley** [comment]
@NetanelBasal lemme know if you like
